package.json

```diff
@@ -31,7 +31,7 @@
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
     "eslint": "^4.15.0",
-    "exquisite-sst": "IagoLast/Exquisite#master",
+    "exquisite-sst": "IagoLast/Exquisite#return-diff-pixels",
```

Comment:
We should we use the master branch (or event a npm version number :D)

Reply:
yep but I dont want to merge stuff in master while the final api is not clear, once this gets approved i´ll merge the exquisite PR and here I´ll point to master.

Finally we should use the 2.0 version of exquisite.

Reply:
The final API is OK for this PR so you can merge it now ;)